src/navigation/_SecondaryNavigationLayout.jsx

```diff
@@ -238,30 +237,21 @@ class SecondaryNavigationLayout extends React.Component {
             onSelect={this.handleCollapsingMenuSelection}
           />
         </div>
-        <OverlayContainer
+        <ContentContainer
           className={cx('content')}
-          overlay={(
-            <Overlay
-              isOpen={isCompact ? compactMenuIsOpen : false}
-              isRelativeToContainer
-              className={cx('overlay')}
-              onRequestClose={this.closeMenu}
+          header={(
```

Comment:
With this configuration the ComponentToolbar won't be covered by the presented overlay. It would have been before. Not sure if this is a big deal or not.

Reply:
Yeah, i think it was weird to cover the toolbar actually. I did this on purpose.